### DISCUSSION TITLE
Css vendor prefixer

### DISCUSSION THREAD

**Discussion by JuliWolf** [nuxt/nuxt#17359] [Questions]
Hello
I trying to migrate to nuxt-bridge and have issue with css vendor autoprefixer

In nuxt 2 all css prefixer were automatically setted for styles in scss files and for inline styles
Now prefixes sets only for css in styles tag but not for inline styles

styles for component is setted in createElement function
```
return createElement('i', {
 ...
  style: {
      maskImage: props.type === 'mask' ? `url(${icon})` : 'none'
  },
  ...
}
```

How i can fix fix problem?

Example mask-image in chrome 
 before (nuxt 2)
![image](https://user-images.githubusercontent.com/43725603/176869575-8287c017-ca71-407a-9ce8-19cd187d0671.png)

 after (nuxt-bridge)
![image](https://user-images.githubusercontent.com/43725603/176869470-c8a45c50-5873-4499-a487-f96c57aa7bc7.png)

**Comment by ojvribeiro** [comment]
I solved this by adding `autoprefixer` as a PostCSS plugin.

In `nuxt.config.ts`, add these lines:

```ts
export default defineNuxtConfig({
  // ...
  postcss: {
    plugins: {
      autoprefixer: {},
    },
  },
})
```
